inc/class-wpseo-options.php

```diff
@@ -2791,192 +2791,195 @@ protected function clean_option( $option_value, $current_version = null, $all_ol
 /*******************************************************************
  * Option: wpseo_ms
  *******************************************************************/
-if ( is_multisite() ) {
```

Comment:
I misunderstood else (thought it's cleaning up instances, not pruning input list). Adjusted in https://github.com/Rarst/wordpress-seo/commit/797d6039205b062cfd30efeb7b422be7b45333a2